runtimes/huggingface/mlserver_huggingface/runtime.py

```diff
@@ -84,14 +84,17 @@ async def predict(self, payload: InferenceRequest) -> InferenceResponse:
         TODO
         """
 
+        # Adding some logging as hard to debug given the many types of input accepted
+        logger.debug("Payload %s", payload)
+
         # TODO: convert and validate?
         kwargs = self.decode_request(payload, default_codec=MultiStringRequestCodec)
 
         args = kwargs.pop("args", [])
         X = kwargs.pop("array_inputs", None)
         if X is not None:
             args = [list(X)] + args
-        prediction = self._model(*args, **kwargs)
+        prediction = self._model(args, **kwargs)
```

Comment:
Actually the *args is necessary as there are some parameters that need to be passed as arg as opposed to as a list. From memory this is the case for the sentiment analysis model, where it would fail if a list is passed https://mlserver.readthedocs.io/en/latest/examples/huggingface/README.html#sentiment-analysis

Reply:
Are there currently errors due to the args not being passed as `*args` instead of a list?

Reply:
So allowing it to be a list allows batch which is needed for alibi-explain

Reply:
See [here](https://github.com/huggingface/transformers/blob/674f750a57431222fa2832503a108df3badf1564/src/transformers/pipelines/base.py#L1019-L1022) which shows if args has multiple values they are ignored.